Autopilot/PathManager/Src/CommsWithTelemetry.cpp

```diff
@@ -56,6 +55,5 @@ bool GetTelemetryCommands(Telemetry_PIGO_t *commands)
         //Indicate that no new commands are available.
         return false;
     }
-    */
     return true;
```

Comment:
Are the changes in this file needed? If not let's exclude this file

Reply:
its removing the comments put by previous Chief